gui/default/index.html

```diff
@@ -330,9 +330,67 @@ <h3 class="panel-title">
         </div>
       </div>
 
-      <!-- First regular row -->
+      <!-- Login form -->
+      <div class="center-block" ng-if="!authenticated">
+        <h3 translate>Authentication required</h1>
 
-      <div class="row">
+        <form ng-submit="authenticatePassword()" class="col-md-6">
+          <div class="row form-group">
+            <label translate for="Username">Username</label>
+            <input id="Username" class="form-control" type="text" ng-model="login.username" tabindex="1" />
+          </div>
+
+          <div class="row form-group">
+            <label translate for="Password">Password</label>
+            <input id="Password" class="form-control" type="password" ng-model="login.password" ng-trim="false" tabindex="2" />
+          </div>
+
+          <div class="row">
+            <div class="col-md-9">
+              <p class="text-danger" ng-if="login.errors.badLogin">
+                Incorrect username and/or password.
+              </p>
+              <p class="text-danger" ng-if="login.errors.failed">
+                Login failed, see server log for details.
+              </p>
+            </div>
+            <div class="col-md-3" style="text-align: right; padding-right: 0">
+              <button type="submit" class="btn btn-default" ng-disabled="login.inProgress" translate>Log in</button>
+            </div>
+          </div>
+        </form>
+
+        <div class="col-md-6" style="text-align: center">
+          <p translate>
+            Or...
+          </p>
+          <button type="button" class="btn btn-primary" ng-click="authenticateWebauthn()" ng-disabled="login.inProgress || isRawIpAddress() || getLocationProtocol() !== 'https'" tabindex="3">
+            <span translate>Log in with WebAuthn</span>
+          </button>
+
+          <p class="text-info" ng-if="isRawIpAddress()" translate>
+            WebAuthn is not supported on raw IP address hosts.
```

Comment:
```suggestion
            WebAuthn is not supported on raw IP addresses. If connected locally using <code>127.0.0.1</code>, replace it with <code>localhost</code> instead.
```

Just a suggestion, as the wording can probably be better. The current message is too high-level for a non-techy user to understand what the problem is about.

Reply:
Yes, I've thought about this too, but I've leaned towards covering this in the docs and adding a link to that instead. I hope the new "reload page at WebAuthn Origin" button I added yesterday can help with this as well.

Reply:
The Docs have their limits though, mainly that they're available in English only, while here the strings are translated into all supported languages. If more changes are coming, let's wait for the final version then!

Reply:
Ah, I see, that's a fair point!

The new button is already pushed in commit https://github.com/syncthing/syncthing/pull/8417/commits/9bc23f51e41b85b7b985c62ab5def326dbe5439b :slightly_smiling_face: Ah, but I see now that this is on the login page, which does not have the button. But I guess we could add one here too - we can even query the back-end for the RP ID (this is already a step in the normal WebAuthn authentication flow, so it won't weaken security) and use that as an even better heuristic for the correct address than just replacing `127.0.0.1` with `localhost`.

Reply:
I've now added a button to reload the page at (probably) the correct address. There'll be some coverage of this in the docs PR too.